Mathlib/CategoryTheory/Monoidal/Rigid/Braided.lean

```diff
@@ -0,0 +1,91 @@
+/-
+Copyright (c) 2024 Gareth Ma. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Gareth Ma
+-/
+import Mathlib.CategoryTheory.Monoidal.Rigid.Basic
+import Mathlib.CategoryTheory.Monoidal.Braided.Basic
+
+/-!
+# Mirror rigid structures via a braiding equivalence.
+-/
+
+open CategoryTheory Category BraidedCategory MonoidalCategory
+
+universe u
+variable {C : Type u} [Category C] [MonoidalCategory C] [BraidedCategory C] {X Y : C}
```

Comment:
```suggestion
variable {C : Type*} [Category C] [MonoidalCategory C] [BraidedCategory C] {X Y : C}
```
since you don't need to mention the universe below, there's no need to even name it.

Reply:
I thought adding universes speed up universe unification? Or does it not make a difference here

Reply:
It doesn't make a difference here. Also `Type*` under the hood does add universes! It's `Type _` which potentially results in universe unification problems (that can take time, and give unexpected results).